ambry-account/src/main/java/com/github/ambry/account/HelixAccountService.java

```diff
@@ -289,7 +289,7 @@ void updateAccountsWithAccountMetadataStore(Collection<Account> accounts, Accoun
     // update operation for all the accounts if any conflict exists. There is a slight chance that the account to update
     // conflicts with the accounts in the local cache, but does not conflict with those in the helixPropertyStore. This
     // will happen if some accounts are updated but the local cache is not refreshed.
-    if (accountInfoMapRef.get().hasConflictingAccount(accounts)) {
+    if (accountInfoMapRef.get().hasConflictingAccount(accounts, false)) {
```

Comment:
I'm wondering if this will be a problem when we switch to mysql primary and Helix secondary.